CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-chapters.bib

```diff
@@ -0,0 +1,243 @@
+% Copyright (C) 2017, The HSF Community White Paper authors, licence CC-BY-4.0.
```

Comment:
There should only be one copy of this file, in roadmap directory, i.e., `HSF-CWP-2017-01_roadmap`, and shared.

Can you take this out please.